### PR TITLE
Expose metrics port

### PR DESCRIPTION
**Pull request by brenner-tobias**
# Proposed Changes

Expose Metrics Server Port and add WebUI URL in Configuration

## Related Issues

#725

## Summary by CodeRabbit

- **New Features**
  - Introduced a metrics web interface for enhanced monitoring.
  - Updated port mapping for improved access to the metrics interface.

- **Configuration Improvements**
  - Reorganized development container settings for better clarity and structure.
  - Updated port configuration for development container to streamline access.
  - Added new network interface entries for metrics in multiple languages.

### CODE REVIEW

File: cloudflared/config.yaml [hunk 1]
```diff
@@ -18,6 +19,10 @@ map:
 options:
   external_hostname: ""
   additional_hosts: []
+ports:
+  36500/tcp: 36500
+ports_description:
+  36500/tcp: Metrics Web interface
```
[the remark this answers is not in the document]
Comment (elcajon) [reply cloudflared/config.yaml:25]: I think we should add the description as [translation](https://developers.home-assistant.io/docs/add-ons/configuration#port-description-translations) to be consistent with the rest of our configuration descriptions.

I would suggest to set the port mapping to "null" to disable port forwarding per default.

**Review by elcajon — APPROVED** [review]
LGTM 👍🏼

**Comment by brenner-tobias** [comment]
There is one issue with the way I implemented it: Since I added the `webui: http://[HOST]:[PORT:36500]/metrics` config, the link for the webui is visible in the add-on page. If you do not actively add the port though, clicking on the link loads a blank website `about:blank#blocked`

We do have three options:

1. Leave as it is right now
2. Remove the webui config, so the link is not shown and it is everyones own responsibility to configure the port and open the metrics link
3. Set the port to be present by default, so there are no issues for most users who do not disable it

I am leaning towards option 2, since this is a niche use case anyways, but I could also live with 3. 

@elcajon what do you think?

**Comment by elcajon** [comment]
I would also tend towards option two, as otherwise the question would arise as to whether you need to forward the port on your router in order to be able to use the add-on and the tunnel at all...

File: cloudflared/config.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ description: "Use a Cloudflare Tunnel to remotely
 url: "https://github.com/brenner-tobias/addon-cloudflared/"
 codenotary: dev@brenner.tech
 init: false
+webui: http://[HOST]:[PORT:36500]/metrics
```
Comment (felipecrs) [inline cloudflared/config.yaml:10]: image

If your intention was to make a button show up here, I think it's missing ingress configuration too.

```yaml
ingress: true
ingress_port: 36500
ingress_stream: true # if websockets are needed, not sure
```

Comment (felipecrs) [reply cloudflared/config.yaml:10]: Nevermind, just saw https://github.com/brenner-tobias/addon-cloudflared/pull/730.

And I agree, I don't think the button should be added indeed for such interface.

BTW it worked fine:

image

Comment (brenner-tobias) [reply cloudflared/config.yaml:10]: Thanks, I think we are good with the current implementation, let me know if you have some suggestions for improvements.

**Pull request merged**